wurst/systems/modes/GameMode.wurst

```diff
@@ -118,6 +118,24 @@ public class GameMode
 
                 modeboard.setTitle("Modes - 00:" + time)
 
+    static function createModeQuest()
```

Comment:
One quest per mode seems rather noisy. All this function should be plural I think to make its function more obvious.